views/bonfire/show.jade

```diff
@@ -143,5 +149,9 @@ block content
                     $('.btn-twitter').attr('href', url);
                 }
         );
+        var MDNlinks = !{JSON.stringify(MDNlinks)};
```

Comment:
I was thinking a separate pr. Easier to merge changes w master on my local copy iteratively than all at once later on. But I'm unaware of whatever the usual protocol is around here. 

Sent from my iPhone

> On Mar 25, 2015, at 19:33, Steve Phillips <notifications@github.com> wrote:
> 
> In views/bonfire/show.jade:
> 
> > @@ -143,5 +149,9 @@ block content
> >                      $('.btn-twitter').attr('href', url);
> >                  }
> >          );
> > +        var MDNlinks = !{JSON.stringify(MDNlinks)};
> @BerkeleyTrue I get what JSON.stringify(MDNlinks) does, but thanks for clarifying that the !{...} syntax is from Jade. I'll look into that.
> 
> —
> Reply to this email directly or view it on GitHub.
>